app/javascript/dashboard/components/widgets/conversation/MessagesView.vue

```diff
@@ -527,6 +534,7 @@ export default {
       />
     </ul>
     <div
+      ref="conversationFooterRef"
```

Comment:
Do we need this change?
```suggestion
      ref="conversationFooterRef"
```